runtime-sdk/src/dispatcher.rs

```diff
@@ -272,64 +271,55 @@ impl<R: Runtime> Dispatcher<R> {
         }
         let tx_auth_info = tx.auth_info.clone();
         let is_read_only = tx.call.read_only;
+        let call = tx.call.clone(); // TODO: Avoid clone.
```

Comment:
what's needed here? dispatch_tx_call needing a copy as "call" as well as the whole tx within the options tx meta?